src/AVFoundation/AVCaptureConnection.cs

```diff
@@ -37,10 +37,15 @@
 namespace AVFoundation {
 	public partial class AVCaptureConnection {
 
-#if !NET
-		[Deprecated (PlatformName.iOS, 7, 0)]
-#else
+#if NET
+		[SupportedOSPlatform ("maccatalyst14.0")]
 		[UnsupportedOSPlatform ("ios7.0")]
+#if IOS
+		[Obsolete ("Starting with ios7.0", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#endif
+		[UnsupportedOSPlatform ("tvos")]
```

Comment:
Personal preference, but I think this line should go up 2 lines with the other 'if NET' supported and not supported (happens in a few other places as well)

Reply:
Right now the generator tries to keep like block near each other, and considers:

```
	[UnsupportedOSPlatform ("ios7.0")]
#if IOS
	[Obsolete ("Starting with ios7.0", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
#endif
```

to be one block, as they are both the output of Deprecated on iOS.